### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by alepev**
Depends on https://github.com/thehyve/delphyne/pull/92

### CODE REVIEW

**Comment by alepev** [comment]
I was wondering also about cleaning up these:
- license code at the top of scripts (since we have a license file anyway)
- relative vs absolute imports: seems to be a mix, and PyCharm doesn't like the absolute ones anymore :(

File: main.py [hunk 1]
```diff
@@ -21,18 +21,15 @@
...
-    # Load configuration
     config = MainConfig(**read_yaml_file(Path(config)))
 
     # Initialize ETL with configuration parameters
     etl = Wrapper(config)
 
-    # TODO: ok to log this here? shall we log it next to wrapper version info?
-    logger.info('ETL version {}'.format(__version__))
-
     # Run ETL
```
Comment (Spayralbe) [inline main.py:33]: I think this comment can be removed as well.

**Comment by Spayralbe** [comment]
The `sample_script.sql` is quite big and relatively complex. Could we maybe use something very small and simple here instead? Like a basic insert into the person table. It would also be good to use the schema placeholders, e.g. `@cdm_schema` instead of `cdm`.

**Comment by Spayralbe** [comment]
> I was wondering also about cleaning up these:
> 
>     * license code at the top of scripts (since we have a license file anyway)
> 
>     * relative vs absolute imports: seems to be a mix, and PyCharm doesn't like the absolute ones anymore :(

I'd say we can remove the license from individual python files.
Good point about the imports. What exactly do you mean with "PyCharm doesn't like the absolute ones anymore"? Personally I'd like to make them all absolute, but as long it's standardized, it's fine with me :)

**Review by Spayralbe — COMMENTED** [review]
So clean ✨🧹

File: main.py (hunk 1 above)
Comment (alepev) [reply main.py:33]: already removed :P it's red

Comment (Spayralbe) [reply main.py:33]: No I mean the comment `# Run ETL`

Comment (alepev) [reply main.py:33]: Sorry, I really misread it. Removed!

**Comment by alepev** [comment]
I've renamed this file for consistency, but I'm still unsure of what we need it for: https://github.com/thehyve/delphyne-template/blob/cleanup/delphyne-template.iml

**Comment by Spayralbe** [comment]
> I've renamed this file for consistency, but I'm still unsure of what we need it for: https://github.com/thehyve/delphyne-template/blob/cleanup/delphyne-template.iml

I don't think we need it? It's an IDE file.

**Comment by alepev** [comment]
> The `sample_script.sql` is quite big and relatively complex. Could we maybe use something very small and simple here instead? Like a basic insert into the person table. It would also be good to use the schema placeholders, e.g. `@cdm_schema` instead of `cdm`.

I added a simpler `sample_script.sql` - maybe too dumb even?
I was wondering though if worth keeping the older one for now (renamed to `observation_period.sql`), it's a very handy piece of code to get the observation period. ideally we might want to translate it into a SQLAlchemy query and store it as a wrapper method, but so far this has been used a lot.. what do you think, shall I make a new issue for this task?

**Comment by alepev** [comment]
> > I've renamed this file for consistency, but I'm still unsure of what we need it for: https://github.com/thehyve/delphyne-template/blob/cleanup/delphyne-template.iml
> 
> I don't think we need it? It's an IDE file.

@MaximMoinat , you added this a while ago, do you still think it's needed and why?

**Comment by MaximMoinat** [comment]
@alepev I am fine with removing. We could use the iml to add some project specific settings if needed, but we don't use it at the  moment.

**Comment by Spayralbe** [comment]
> > The `sample_script.sql` is quite big and relatively complex. Could we maybe use something very small and simple here instead? Like a basic insert into the person table. It would also be good to use the schema placeholders, e.g. `@cdm_schema` instead of `cdm`.
> 
> I added a simpler `sample_script.sql` - maybe too dumb even?
> I was wondering though if worth keeping the older one for now (renamed to `observation_period.sql`), it's a very handy piece of code to get the observation period. ideally we might want to translate it into a SQLAlchemy query and store it as a wrapper method, but so far this has been used a lot.. what do you think, shall I make a new issue for this task? (and move the code to the issue, so I can delete it from the template)

I think the current script is fine. It's just for showing an example of how to run a query from a sql file.

Regarding `observation_period.sql`, it would be useful to store these indeed, but I think this is the right repo for that. We could create a subfolder like `sql/general/` for storing such sql files. A sub-task of https://github.com/thehyve/delphyne/issues/23 could then be turn these into SQLAlchemy queries.

**Comment by alepev** [comment]
✅  Removed IML file
✅  Moved observation period script to subfolder for later refactoring

**Pull request merged**